test/cqlpy/test_describe.py

```diff
@@ -496,38 +496,60 @@ def is_scylla(cql):
 
 # Test that 'DESC INDEX' contains create statement of index
 def test_desc_index(cql, test_keyspace):
-    with new_test_table(cql, test_keyspace, "a int primary key, b int, c int") as tbl:
+    with new_test_table(cql, test_keyspace, "a int primary key, b vector<float, 5>, c vector<float, 5>") as tbl:
         tbl_name = get_name(tbl)
         create_idx_b = f"CREATE INDEX ON {tbl}(b)"
         create_idx_c = f"CREATE INDEX named_index ON {tbl}(c)"
         # Only Scylla supports local indexes
         has_local = is_scylla(cql)
+
+        # Cassandra inserts a space between the table name and parentheses,
```

Comment:
One of the nice things about the test/cqlpy framework is that there is no major advantage to testing 7 different things in one test function instead of having 7 test functions.

Please leave the original test function unmodified and add a new function testing the  INDEX .. USING (custom index) feature.

Having a separate function makes it easier to understand, gives you a place to add a comment explaining what is being tested and refer to some issue, etc., and if one day we have a regression, we know exactly what failed.

Imagine one day in the future we rename "opensearch_vector_index" and suddenly this test fails. It is nicer to see that the "custom index" test failed but the original index test still passes, and more easily understand what went wrong.